Source/WebCore/platform/gstreamer/GStreamerQuirkRialto.cpp

```diff
@@ -87,6 +88,10 @@ GstElement* GStreamerQuirkRialto::createAudioSink()
 
 GstElement* GStreamerQuirkRialto::createWebAudioSink()
 {
+    if (GstElement* sink = webkitAudioSinkNew()) {
+        return sink;
+    }
+
```

Comment:
If we do this, then why do we need to rest of the function? Just in case this fails?

Reply:
There is an env inside webkit audio sink that we can use to select which sink to use. With this we are able to enable/disable audio mixer without changing webkit:
Source/WebCore/platform/graphics/gstreamer/WebKitAudioSinkGStreamer.cpp
```
    const char* value = g_getenv("WEBKIT_GST_ENABLE_AUDIO_MIXER");
    if (value && !strcmp(value, "1")) {
        if (!GStreamerAudioMixer::isAvailable()) {
            GST_WARNING("Internal audio mixing request cannot be fulfilled.");
            return false;
        }

```